### PR TITLE
Theme BrightNights up a bit

### PR DESCRIPTION
**Pull request by Coolthulhu**
Includes #20469

BN is supposed to be a light "adventure" mod, not just sci-fi one.
Here this means easy crafting of supplies such as ammo and meds.

Another theme I'd like to capitalize on is "drugs as potions". Take damage often, heal up with bandages, kill the pain with opiates and ethanol.

And a more personal thing: moving the action place somewhere north. At the moment it means just less guns, more alcohol and lower average temperature, but keeps all the moose, so it probably looks rather Canadian. Eventually I want to move it to Eastern Europe: improvised guns (but almost no real ones), concrete living blocks, cold weather, alcohol, opiates, poverty, pollution, hopelessness, high suic... reckless behavior among the population.

Changes here:
* Added dependence of the mod on craftable gunpowder
* Reintroduced the unrealistic recipes for bandages and torches out of vodka into the mod
* Regional settings added to mod: colder, less military, more liquor stores

### CODE REVIEW

**Comment by AMurkin** [comment]
Yeah, 'Cold Russian nights' mod :D.
Sounds interesting.

**Comment by Rivet-the-Zombie** [comment]
>alcohol, opiates, poverty, pollution, hopelessness, high suic...

This part of it still sounds like the USA.

**Comment by kevingranade** [comment]
Rebase?  It's still got the nested crafting lists stuff in it.

**Comment by Coolthulhu** [comment]
Rebased

File: data/mods/BrightNights/regional_map_settings.json [hunk 1]
```diff
@@ -0,0 +1,167 @@
+[
+   {
+      "type": "region_settings",
```
Comment (kevingranade) [inline data/mods/BrightNights/regional_map_settings.json]: So nice that this is finally getting some use.

**Pull request merged**